src/extension.ts

```diff
@@ -139,6 +141,9 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     },
     () => {
       const msg = (() => {
+        const proxy =
+          `See https://scalameta.org/metals/docs/editors/vscode.html#http-proxy for instructions ` +
+          `if you are using a corporate HTTP proxy.`;
```

Comment:
```suggestion
          `if you are using an HTTP proxy.`;
```

Not all proxies are corporate (e.g. universities)

Reply:
Good point 👍